shims/shims_for_IE.js

```diff
@@ -18,9 +18,9 @@ require('html5-history-api');  // HTML5 History: IE9.
 if (!Object.hasOwnProperty('name')) {
   Object.defineProperty(Function.prototype, 'name', {
     get: function getName() {
-      const matches = this.toString()
+      var matches = this.toString()
```

Comment:
Cool, thanks. What about the duplicate .eslintignore thing I mentioned above - does what I said make sense?